### PR TITLE
feat: wallet doc updates

### PR DESCRIPTION
**Pull request by yknl**
Updates the wallet documentation for v3.1.0 as well as some clean up.

### CODE REVIEW

File: src/pages/stacks-wallet/overview.md [hunk 1]
```diff
@@ -5,40 +5,37 @@ description: Learn about cryptocurrency wallets, STX, and the Stacks wallet.
...
 ## Introduction
 
-If you are, or plan to be a Stacks token holder, you need to think about how you
-can manage your Stacks (STX) tokens. How can you review your token balance? How can send or
-receive your tokens? STX tokens are a cryptocurrency and you manage your tokens through a **cryptocurrency wallet**.
-This overview introduces basic wallet terminology and concepts.
+The Stacks Wallet is the application used to manage your Stacks (STX) token holdings. You can
+use it to view your token balance, send and receive tokens. This overview will introduce some
+basic concepts and terminology. If you have a good understanding of cryptocurrencies, you can
+skip to the [installation page](https://docs.blockstack.org/stacks-wallet/install).
```
Comment (agraebe) [inline src/pages/stacks-wallet/overview.md:11]: let's link to "/stacks-wallet/install" instead

File: src/pages/stacks-wallet/usage.md [hunk 2]
```diff
@@ -292,10 +292,10 @@ Due to technical compatibility issues, we are currently unable to fully support
 
 -> The wallet only supports Trezor One, Ledger Nano S, and Ledger Blue. Other wallets, for example, the Trezor Model T, are not supported. If you have questions about wallet support, please [contact Blockstack support](emailto:support@blockstack.org).
 
-The current Stacks Wallet ([v3.0.0](https://github.com/blockstack/stacks-wallet/releases/tag/v3.0.0)) supports only the Trezor One device, and only up to firmware [version 1.8.3](https://wiki.trezor.io/Firmware_changelog). If you have a newer firmware, you will have to downgrade by [following instructions described on this page](https://wiki.trezor.io/Firmware_downgrade).
+The current Stacks Wallet ([v3.1.0](https://github.com/blockstack/stacks-wallet/releases/tag/v3.0.0)) supports only the Trezor One device, and only up to firmware [version 1.8.3](https://wiki.trezor.io/Firmware_changelog). If you have a newer firmware, you will have to downgrade by [following instructions described on this page](https://wiki.trezor.io/Firmware_downgrade).
```
Comment (agraebe) [inline src/pages/stacks-wallet/usage.md:295]: the link should be updated to v3.1.0

**Review by agraebe — APPROVED** [review]
thanks! I think 2 small updates are needed. I'm approving to allow merging once updated :)

**Comment by markmhx** [comment]
These changes generally look good to me. Thanks @yknl 🙏

**Pull request merged**